app/models/tournament.rb

```diff
@@ -3,4 +3,6 @@ class Tournament < ApplicationRecord
   belongs_to :event, optional: true
   has_many :players
   has_many :matchups, through: :players
+  validates :pairing_system, :event_id, presence: true
```

Comment:
Not sure about validating the event_id here. Might be useful to create a tournament that isn't on cross-tables